monai/bundle/utils.py

```diff
@@ -221,6 +221,7 @@ def load_bundle_config(bundle_path: str, *config_names: str, **load_kw_args: Any
                 raise ValueError(f"Cannot find config file '{full_cname}'")
 
             ardata = archive.read(full_cname)
+            cdata = {}
```

Comment:
May I ask the change here is for what?

Reply:
We know that `full_cname` will end with either JSON or YAML file extensions, but this assumption isn't available to checkers, so we should defined the variable before the `if-elif` so that it is clear it will always exist, even if the two code paths don't trigger.